doc/magic_cards_notes.md

```diff
@@ -1908,6 +1908,15 @@ This mode is divided into four states: off (pre-write), on (on restore), don’t
 If you use it, please enter the pre-write mode first. At this time, write the full card data.
 After writing, set it to on. At this time, after writing the data, the first time you read the data just written, the next time you read It is the pre-written data. All modes support this operation. It should be noted that using any block to read and write in this mode may give wrong results.
 
+*This is very starnge and fuzzed descripotion. I don`t understand how it can work, if "using any block to read and write in this mode may give wrong result" ? Maybe this mode not work for old card too?*
+
```

Comment:
Rewrite this,    it doesnt belong in this document.       

- Only write about the subject at hand.
- Some spelling mistakes in your PR.  Run a spell check please.

Reply:
Cleaning and small refactoring.